lua/config/lazy.lua

```diff
@@ -23,6 +23,8 @@ require("lazy").setup({
     { import = "lazyvim.plugins.extras.lang.python" },
 
     { import = "lazyvim.plugins.extras.lang.tailwind" },
+    { import = "lazyvim.plugins.extras.lang.typescript" },
+    { import = "lazyvim.plugins.extras.lang.yaml" },
```

Comment:
wait why did you add these? the yaml one is already there on like 30 and we don't need to import the typescript one because we are doing our own typescript setup in that typescript.lua file